### PR TITLE
add holiday eBee backstory

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

**Comment by xmulligan** [comment]
I was trying to add a few different traditions, so I added Christmas, Chinese new years, and Hanukkah. I'm happy to add others, those were just the first ones I came up with

**Comment by qmonnet** [comment]
Oooh so that's why I didn't had the references :sweat_smile: I thought they were all local Christmas traditions :facepalm:. OK never mind me then :)

File: README.md [hunk 1]
```diff
@@ -212,6 +212,17 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
        What do you need a backstory for when you're resting on a hamoc under a gentle breeze, on the warm and sunny beaches of a Caribbean island? There's no place like the seashore to meet with friends and discuss new networking designs (Oh wait, I'm hearing that Skiing eBee disagrees with this statement). Caribbean eBee invites you for some fun and relaxation. Take a seat next to her. Sip your honey-rum-coconut juice. Listen to the lulling sound of the waves. Enjoy.
    </td>
   </tr>
+  <tr>
+   <td> <img src="" alt="Holiday bee">
```
Comment (xmulligan) [inline README.md:216]:
```suggestion
   <td> <img src="images/holiday-bee.png" alt="Holiday bee">
```

**Comment by qmonnet** [comment]
Looks like some of the picture became transparent. Did you do “white to alpha”?

**Comment by xmulligan** [comment]
I did color to alpha. I thought that was how you removed the white background 😅

File: README.md [hunk 2]
```diff
@@ -212,6 +212,17 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
...
    </td>
   </tr>
+  <tr>
+   <td> <img src="images/holiday-bee.png" alt="Holiday bee">
+   </td>
+   </td>
+   <td>#21: Holiday eBee
+     <br />
+     <br />
+       Backstory:
```
Comment (qmonnet) [inline README.md:222]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -212,6 +212,17 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
        What do you need a backstory for when you're resting on a hamoc under a gentle breeze, on the warm and sunny beaches of a Caribbean island? There's no place like the seashore to meet with friends and discuss new networking designs (Oh wait, I'm hearing that Skiing eBee disagrees with this statement). Caribbean eBee invites you for some fun and relaxation. Take a seat next to her. Sip your honey-rum-coconut juice. Listen to the lulling sound of the waves. Enjoy.
    </td>
   </tr>
+  <tr>
+   <td> <img src="images/holiday-bee.png" alt="Holiday bee">
+   </td>
+   </td>
+   <td>#21: Holiday eBee
```
Comment (qmonnet) [inline README.md:219]:
```suggestion
   <td>#22: Holiday eBee
```

**Review by qmonnet — CHANGES_REQUESTED** [review]

**Comment by qmonnet** [comment]
> I did color to alpha. I thought that was how you removed the white background sweat_smile

It works for some pictures, but if there's white in the drawing it will remove it, too. I'm using the selection tool to select the background only, then turn it to alpha or simply delete the selection. I'm far from being an expert with graphic tools though, so that's just my two pennies. Happy to help for this one if you have the original picture somewhere

**Comment by xmulligan** [comment]
I pulled the image right from the issue https://github.com/isovalent/ebeedex/issues/43. Would be great if you had a better way to do it :)

**Comment by xmulligan** [comment]
updated now

**Review by qmonnet — APPROVED** [review]
Looks all good

**Pull request merged**